### ISSUE TITLE
Build failed for UE4.21

### ISSUE THREAD

**Issue by atifkarim** [unrealcv/unrealcv#156]
- Operating System: Windows 10, 64 bit
- UE4 Version: 4.21
- UnrealCV Version: master, fast
- Client (python2, 3 or matlab): python3
- Problem Description:

I have tried to build UnrealCV for UE4.21 but it has failed. Both master and fast branch I have tried but an error has occurred. Can you please inform whether there is any support for this or not?

**Comment by qiuwch** [comment]
Can you paste your error here? Fast branch supports 4.21. Master branch is not compatible with 4.21.

**Comment by atifkarim** [comment]
@qiuwch I have uploaded here the log file and the screenshot.
[Log.txt](https://github.com/unrealcv/unrealcv/files/2805349/Log.txt)
[UBT-UE4Editor-Win64-Development.txt](https://github.com/unrealcv/unrealcv/files/2805350/UBT-UE4Editor-Win64-Development.txt)
![ue4 21_error_for_ucv](https://user-images.githubusercontent.com/32655256/51872786-f969cc80-235a-11e9-849a-43da1ed9f07e.PNG)

**Comment by qiuwch** [comment]
I fixed the compile issue for 4.21, but did not have time to rigorously test the features yet.

The reason for the failure is the UE4 build tool has a different logic for discovering header files. The major change is in the file `UnrealCV.build.cs`.

I am traveling these two days and will run some tests after got back.

The command I used for testing the build is `E:\\UE_4.21\\Engine\\Build\\BatchFiles\\RunUAT.bat BuildPlugin -plugin=C:/qiuwch/workspace/unrealcv/UnrealCV.uplugin -package=C:/qiuwch/workspace/unrealcv/build -rocket -targetplatforms=Win64`
